### ISSUE TITLE
[BUG] Readme is outdated 

### ISSUE THREAD

**Issue by polaroi8d** [brimblehq/brimble#30]
I tried to run the application locally, but when the REDME.md says "open client and API folder in the packages" I didn't find anything. I did not find any `.env.example` files. How can I move forward?

**Comment by klefcodes** [comment]
Yeah @polaroi8d, thanks for the opened ticket. We will update the readme accordingly 😉

**Comment by realitystevens** [comment]
@klef-dev README hasn't been updated yet
